src/main/java/duke/WrongInputException.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+/**
+ * This class is used to represent an exception thrown when a Wrong Input is given
+ */
+public class WrongInputException extends Exception {
+
+    /**
+     * Constructor for creating a WrongInputTask exception
+     * @param taskExceptionMessage What is the error or invalid format
+     * @param solution  What is the recommended solution
```

Comment:
```suggestion
     * @param solution The recommended solution
```

Reply:
Phrasing seems more like a question rather than a statement you could consider rephrasing this